client/devdocs/design/blocks.jsx

```diff
@@ -47,6 +47,7 @@ import PostStatus from 'blocks/post-status/docs/example';
 import ReaderAuthorLink from 'blocks/reader-author-link/docs/example';
 import ReaderSiteStreamLink from 'blocks/reader-site-stream-link/docs/example';
 import ReaderFullPostHeader from 'blocks/reader-full-post/docs/header-example';
+import ReaderSearchStreamHeader from 'reader/search-stream/docs/search-stream-header';
```

Comment:
i put it in docs because i wanted there to be a way for you to see the component.
As discussed in slack I've started using the feature flag so that you can demo the component without it being in devdocs